roles/openshift_aws/defaults/main.yml

```diff
@@ -114,6 +126,9 @@ openshift_aws_node_group_config:
     wait_for_instances: True
     termination_policy: "{{ openshift_aws_node_group_termination_policy }}"
     replace_all_instances: "{{ openshift_aws_node_group_replace_all_instances }}"
+    elbs:
+    - "{{ openshift_aws_clusterid }}-{{ openshift_aws_node_group_type}}-internal"
```

Comment:
This does not align with variable openshift_aws_elb_name

Looks like we need to do more refactoring.

We should create a new variable like this:

```
openshift_aws_elb_name_dict:
  infra:
    - "{{ openshift_aws_elb_basename }}-infra-internal"
    - "{{ openshift_aws_elb_basename }}-infra-external"
  masters:
    ...
  something_else:
```

And reference that dictionary on the elbs line above as such:
```
elbs: "{{ openshift_aws_elb_name_dict[openshift_aws_node_group_type] }}"
```

This will allow users to re-use existing elbs as they see fit.

Reply:
@michaelgugino, thanks, let me give this a whirl.